app/services/pupil_premium_importer.rb

```diff
@@ -30,8 +30,8 @@ def update_school_premium(row)
     school = School.find_by(urn: urn)
     logger.info "Could not find school with URN #{urn}" and return unless school
 
-    total_pupils = row.fetch("Number of pupils on roll (7)")
-    eligible_pupils = row.fetch("Total number of pupils eligible for the Deprivation Pupil Premium")
+    total_pupils = row.fetch("Number of pupils on roll (7)").delete(",").to_i
```

Comment:
Do we need to worry about spaces too?

Reply:
I don't think so... This is excel's fault, and while it puts commas in numbers, I don't think it's quite pathological enough to put spaces in. On the other hand... 🤷